src/components/Form.vue

```diff
@@ -20,10 +20,13 @@
 </template>
 
 <script>
+import { mapActions } from "vuex";
```

Comment:
Tutaj dodaj mapGetters

Reply:
dlaczego mapGetters? nie korzystam w tym komponencie z getterów. Może dodać mapState, żeby wyświetlić textInfo czy zamienić textInfo na gettera i wtedy go wyświetlnić?

Reply:
@Jakub-Klejczyk  jak to nie korzystasz z getterów, a to co to jest? 
<img width="647" alt="Zrzut ekranu 2022-08-21 o 14 30 03" src="https://user-images.githubusercontent.com/25270903/185790978-01a2541f-d417-4d20-b41e-c284ff9e8a1b.png">

Reply:
<img width="466" alt="Zrzut ekranu 2022-08-21 o 14 30 54" src="https://user-images.githubusercontent.com/25270903/185790990-033fcd4d-aa4b-44b6-936b-e7ade58d1038.png">
A to ?